ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1264,6 +1264,11 @@ namespace OrdinalAxis {
                         xData: (series.xData as any).slice(),
                         chart: chart,
                         groupPixelWidth: series.groupPixelWidth,
+                        data: series.type === 'timeline' ? series.data : null,
```

Comment:
I would rather avoid type-specific fixes in the code. Did you try simply `data: series.data`? 

PS: To check for series type, use `series.is('name')` because it also checks the inheritance (eg `series.is('column')` will return true for columnrange, boxplot, bullet etc series)